Makefile

```diff
@@ -18,3 +18,10 @@ all:
 	go build `go list ./... | grep -v 'vendor'`
 
 include release-tools/build.make
+
+# Check contextual logging.
+.PHONY: logcheck
+test: logcheck
+logcheck:
+	go install sigs.k8s.io/logtools/logcheck@v0.8.1
+	PATH=$$(go env GOPATH)/bin:$$PATH logcheck -check-contextual ./...
```

Comment:
If some developer runs "make test", this will overwrite their `$GOPATH/bin/logcheck`, which will be surprising.

A better solution is to install into a temporary directory (`GOBIN=<tmp dir> go install ...`) and invoke the tool there with an absolute path (no need to set `PATH`).

Remember to delete the temporary directory with `trap EXIT`.

This implies doing everything in a single shell. A verify-logcheck.sh script might be easier than doing it all inline in the Makefile.

Reply:
@pohly I have created the `verify-logcheck.sh` script and updated the Makefile to execute this script. For the installation of logcheck, a temp dir is used, and it is set up to be deleted at the end of the script execution using `trap EXIT`.

Could you please review it again? Thank you.@pohly I have created the `verify-logcheck.sh` script and updated the Makefile to execute this script. For the installation of logcheck, a temp dir is used, and it is set up to be deleted at the end of the script execution using `trap EXIT`.

Could you please review it again? Thank you.